bin/polkadot

```diff
@@ -0,0 +1 @@
+../../../everything-trappist/trappist/bin/polkadot
```

Comment:
We should remove bin folder and just let the developer choosing their own bin locations.

Reply:
Will do